applications/new_schema.py

```diff
@@ -71,6 +76,34 @@ def get_node(cls, info, id):
         return super().get_node(info, id)
 
 
+class UpdateBerthApplication(graphene.ClientIDMutation):
+    class Input:
+        id = graphene.ID(required=True)
+        customer_id = graphene.ID(required=True)
```

Comment:
🤔 if we want to make it a more general update mutation, we would need to allow for passing only those values, that the frontend wants to update, i.e. only the `id` of the application should be required. other fields should be updated if there was input.

Reply:
hm, could you it the way profile does it with [the `InputObjectType`](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/schema.py#L272)? we could refactor inputs in resources later on to follow the same convention

Reply:
Sure. There's also something like that in [resources](https://github.com/City-of-Helsinki/berth-reservations/blob/da62f2ea6868a2f9eda35c66166d84c68b5fbec8/resources/schema.py#L464).

For now, I thought it would make sense to keep the `customer_id` as required since it's the only "updatable" field and it wouldn't make sense to run an empy mutation which just receives the application id. We can either ditch the required right away or add (yet) another `TODO` to remind removing the required once we start allowing other fields.

Reply:
OK, fair enough! let's keep the input arguments as is and refactor when the need arises